deepinv/__init__.py

```diff
@@ -48,10 +48,14 @@
 
 __all__ += ["sampling"]
 
-from deepinv.loss import metric as metric
+from deepinv.loss import metric
 
 __all__ += ["metric"]
 
+__all__ += ["unfolded"]
+
+from deepinv import unfolded

```

Comment:
Reverse the two lines?
```suggestion
from deepinv import unfolded

__all__ += ["unfolded"]
```